src/sentry/tasks/post_process.py

```diff
@@ -1118,6 +1123,12 @@ def process_code_mappings(job: PostProcessJob) -> None:
             org_slug = org.slug
             next_time = timezone.now() + timedelta(hours=1)
 
+            # php automatic code mappings currently in LA
+            if "php" in event.data["platform"] and not features.has(
```

Comment:
```suggestion
            if event.data["platform"].startswith("php") and not features.has(
```
`in` works, but could be potentially unnecessarily broad. AFAIK platform ID is always `"{lang}"` or `"{lang}-{framework}"`. 

The valid PHP ones are: https://github.com/getsentry/sentry/blob/49888e050b434e0073adbe434e7738f1e7ea730e/src/sentry/models/project.py#L119-L121